### PR TITLE
Fixed typos in docs/guides

### PR DESCRIPTION
**Pull request by Mr-Destructive**
Corrected several typos and grammatical errors in the `docs/guide` folder.

Please review once for any further corrections.
Thank you :+1:

### CODE REVIEW

File: docs/guides/routes.md [hunk 1]
```diff
@@ -48,24 +48,22 @@ For more information on `crow::response` go [here](../../reference/structcrow_1_
...
+Your class should look like the following:
 ```cpp
 class a : public crow::returnable 
 {
     a() : returnable("text/plain"){};
-
     ...
     ...
     ...
-
```
Comment (luca-schlecker) [inline docs/guides/routes.md:68]: I'd prefer (including the empty lines):
```

...

```

**Review by luca-schlecker — CHANGES_REQUESTED** [review]
Looking good otherwise. 👍

**Review by luca-schlecker — APPROVED** [review]
Thank you for your submission. 🎉

File: docs/guides/multipart.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The structure of a multipart request is typically consistent of:<br>
 Crow supports multipart requests and responses though `crow::multipart::message`.<br>
 A message can be created either by defining the headers, boundary, and individual parts and using them to create the message. or simply by reading a `crow::request`.<br><br>
 
-Once a multipart message has been made, the individual parts can be accessed throught `mpmes.parts`, `parts` is an `std::vector`, so accessing the individual parts should be straightforward.<br>
+Once a multipart message has been made, the individual parts can be accessed throughout `mpmes.parts`, `parts` is an `std::vector`, so accessing the individual parts should be straightforward.<br>
```
Comment (The-EDev) [inline docs/guides/multipart.md:19]: @luca-schlecker @Mr-Destructive This is acutally meant to be "through", I'll make another commit fixing it.

Comment (Mr-Destructive) [reply docs/guides/multipart.md:19]: I'm sorry I interpreted the meaning wrongly.

Comment (luca-schlecker) [reply docs/guides/multipart.md:19]: Oh, I am sorry... I looked at it and agreed with him because I thought it was a list... My bad.

Comment (The-EDev) [reply docs/guides/multipart.md:19]: no worries, it was my typo to begin with 😅. I'll fix it asap

**Pull request merged**